### DISCUSSION TITLE
react-zxing

### DISCUSSION THREAD

**Discussion by rozkurt54** [community/community#38404] [Other Feature Feedback, Questions, & Ideas]
Hi,

I want to use react-zxing,

when the install react-zxing by the npm

some files are not installed, 

I am looking node_modules/@zxing folder, its only including /library and  /text-encoding folder.

when, start the project,
its failed 
WARNING in ./node_modules/@zxing/library/esm/browser.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '/home/ramazan/donmeznew/node_modules/@zxing/src/browser.ts' file: Error: ENOENT: no such file or directory, open '/home/ramazan/donmeznew/node_modules/@zxing/src/browser.ts'
vs. some error messages.

**Comment by DSDmark** [comment]
Hi @rozkurt54,

I'm not sure, but this could help you:-
https://github.com/zxing-js/library/issues/298

And also I read some discussion about that It's said try different version.

Or if you want that library alone, then you install that:- 
https://www.npmjs.com/package/@zxing/library

And also you could try this if its work. Go package.json and in script area write this:-
```"start": "GENERATE_SOURCEMAP=false react-scripts start",```

**Comment by MostafaWaleed0** [comment]
The issue with missing files in the `node_modules/@zxing` folder could be due to a number of reasons, such as network connectivity issues, incorrect version of the package, or compatibility issues with your project setup.

1. Check Network Connectivity: Make sure your internet connection is stable and that you have access to the npm registry.
2. Reinstall the Package: Try uninstalling and reinstalling the `react-zxing` package to see if it resolves the issue.
```powershell
npm uninstall react-zxing
npm install react-zxing
```
3. Check Package Version: Make sure that you have installed the latest version of the package. You can check the available versions in the npm registry and choose the latest version that is compatible with your project setup.
```powershell
npm view react-zxing versions
npm install react-zxing@<latest-version>
```
4. Check for Compatibility Issues: If you are still experiencing issues, check for compatibility issues with your project setup. You can check the package's documentation or the GitHub repository for information on compatibility and installation requirements.

5. Contact Support: If the issue persists, consider reaching out to the package's support team or the community for help.
